plugins/UM3NetworkPrinting/src/Models/Http/ClusterPrintJobStatus.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, created_at: str, force: bool, machine_variant: str, name: str
                  build_plate: Union[Dict[str, Any], ClusterBuildPlate] = None,
                  compatible_machine_families: List[str] = None,
                  impediments_to_printing: List[Union[Dict[str, Any], ClusterPrintJobImpediment]] = None,
+                 preview_url = None,
```

Comment:
type is optional string

Reply:
oops, that slipped through, let me fix that real quick

Reply:
Resolved with #ac5213d